NoSuchMethodError occurs in gcs_wait operator from version 0.10.0

NoSuchMethodError occurs in gcs_wait operator from version 0.10.0.
And continue to wait after an error occurs.

## Sample
sample.dig

```
+gcs_wait:
  gcs_wait>: digdag-gcs_wait-test/tmp.txt
```

run with 0.9.42

```
$ digdag --version
0.9.42

$ digdag run sample.dig --rerun
2021-03-12 11:17:17 +0900: Digdag v0.9.42
2021-03-12 11:17:19 +0900 [WARN] (main): Reusing the last session time 2021-03-12T00:00:00+00:00.
2021-03-12 11:17:19 +0900 [INFO] (main): Using session /Users/katsuya.tajima/ghq/github.com/katsuyan/digdag-example/gcs_wait/.digdag/status/20210312T000000+0000.
2021-03-12 11:17:19 +0900 [INFO] (main): Starting a new session project id=1 workflow name=sample session_time=2021-03-12T00:00:00+00:00
2021-03-12 11:17:20 +0900 [INFO] (0018@[0:default]+sample+gcs_wait): gcs_wait>: digdag-gcs_wait-test/tmp.txt
Success. Task state is saved at /Users/katsuya.tajima/ghq/github.com/katsuyan/digdag-example/gcs_wait/.digdag/status/20210312T000000+0000 directory.
  * Use --session <daily | hourly | "yyyy-MM-dd[ HH:mm:ss]"> to not reuse the last session time.
  * Use --rerun, --start +NAME, or --goal +NAME argument to rerun skipped tasks.
```

run with 0.10.0

```
$ digdag --version
0.10.0

$ digdag run sample.dig --rerun
2021-03-12 11:17:30 +0900: Digdag v0.10.0
2021-03-12 11:17:32 +0900 [WARN] (main): Reusing the last session time 2021-03-12T00:00:00+00:00.
2021-03-12 11:17:32 +0900 [INFO] (main): Using session /Users/katsuya.tajima/ghq/github.com/katsuyan/digdag-example/gcs_wait/.digdag/status/20210312T000000+0000.
2021-03-12 11:17:32 +0900 [INFO] (main): Starting a new session project id=1 workflow name=sample session_time=2021-03-12T00:00:00+00:00
2021-03-12 11:17:32 +0900 [INFO] (0018@[0:default]+sample+gcs_wait): gcs_wait>: digdag-gcs_wait-test/tmp.txt
2021-03-12 11:17:32 +0900 [ERROR] (0018@[0:default]+sample+gcs_wait): Task failed with unexpected error: com.google.api.client.googleapis.services.json.AbstractGoogleJsonClient$Builder.setBatchPath(Ljava/lang/String;)Lcom/google/api/client/googleapis/services/AbstractGoogleClient$Builder;
java.lang.NoSuchMethodError: com.google.api.client.googleapis.services.json.AbstractGoogleJsonClient$Builder.setBatchPath(Ljava/lang/String;)Lcom/google/api/client/googleapis/services/AbstractGoogleClient$Builder;
        at com.google.api.services.storage.Storage$Builder.setBatchPath(Storage.java:11057)
        at com.google.api.services.storage.Storage$Builder.<init>(Storage.java:11036)
        at io.digdag.standards.operator.gcp.GcsClient.client(GcsClient.java:38)
        at io.digdag.standards.operator.gcp.GcsClient.client(GcsClient.java:21)
        at io.digdag.standards.operator.gcp.BaseGcpClient.<init>(BaseGcpClient.java:30)
        at io.digdag.standards.operator.gcp.GcsClient.<init>(GcsClient.java:28)
        at io.digdag.standards.operator.gcp.GcsClient$Factory.create(GcsClient.java:70)
        at io.digdag.standards.operator.gcp.BaseGcsOperator.run(BaseGcsOperator.java:27)
        at io.digdag.standards.operator.gcp.BaseGcpOperator.runTask(BaseGcpOperator.java:25)
        at io.digdag.util.BaseOperator.run(BaseOperator.java:35)
        at io.digdag.core.agent.OperatorManager.callExecutor(OperatorManager.java:365)
        at io.digdag.cli.Run$OperatorManagerWithSkip.callExecutor(Run.java:707)
        at io.digdag.core.agent.OperatorManager.runWithWorkspace(OperatorManager.java:298)
        at io.digdag.core.agent.OperatorManager.lambda$runWithHeartbeat$2(OperatorManager.java:151)
        at io.digdag.core.agent.LocalWorkspaceManager.withExtractedArchive(LocalWorkspaceManager.java:25)
        at io.digdag.core.agent.OperatorManager.runWithHeartbeat(OperatorManager.java:149)
        at io.digdag.core.agent.OperatorManager.run(OperatorManager.java:132)
        at io.digdag.cli.Run$OperatorManagerWithSkip.run(Run.java:689)
        at io.digdag.core.agent.MultiThreadAgent.lambda$null$0(MultiThreadAgent.java:132)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
^C%
```

--- a/digdag-standards/build.gradle
+++ b/digdag-standards/build.gradle
@@ -40,7 +40,6 @@ dependencies {
     // gcs
     compile ('com.google.apis:google-api-services-storage:v1-rev20190910-1.30.3') {
         exclude group: 'com.google.guava', module: 'guava-jdk5'
-        exclude group: 'com.google.api-client', module: 'google-api-client'
     }
 
     // kubernetes

